test/README.md

```diff
@@ -13,45 +13,62 @@ with nested virtualization, refer to
 [this Fedora guide](https://docs.fedoraproject.org/en-US/quick-docs/using-nested-virtualization-in-kvm/index.html)
 for more details and recommendations on ensuring it is enabled correctly.
 
-## Introduction
+## Preparation and general invocation
 
-Before running the tests, ensure Cockpit has been built where the test suite
-expects to find it (do NOT run the build step as root):
+*Warning*: Never run the build, test, or any other command here as root!
 
-    $ ./test/image-prepare
+You first need to build cockpit, and install it into a VM:
 
-To run the integration tests run the following (do NOT run the integration tests
-as root):
+    $ test/image-prepare
 
-    $ ./test/common/run-tests
+This uses the default OS image, which is currently Fedora 35. See `$TEST_OS`
+below how to select a different one.
 
-The tests will automatically download the VM images they need, so expect
-that the initial run may take a couple of hours (there are quite a few
-images to retrieve for different scenario tests).
+In most cases you want to run an individual test:
+
+    $ test/verify/check-metrics TestCurrentMetrics.testCPU
+
```

Comment:
I would update this slightly.
I would but there `... for example test/verify...` now it sounds like there is only this one. 

Also would be worth mentioning that you can use `--list` to see the list of all available tests in given file. (Tricky now to see where you got `TestCurrentMetrics.testCPU`)

Reply:
Thanks. Updated for both.